app/src/main/java/com/fstyle/androidtrainning/screen/tabfavorite/TabFavoriteFragment.java

```diff
@@ -1,15 +1,20 @@
 package com.fstyle.androidtrainning.screen.tabfavorite;
 
+import android.content.Context;
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v7.widget.GridLayoutManager;
 import android.support.v7.widget.RecyclerView;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import com.fstyle.androidtrainning.MainApplication;
 import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.data.local.InsertListMovieToDatabase;
+import com.fstyle.androidtrainning.data.local.InsertMovieToDatabase;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.data.local.InsertMovieToDatabase. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports